versions-ecosystem.cfg

```
@@ -9,8 +9,8 @@ plone.app.blocks = 7.0.0
 plone.app.debugtoolbar = 1.3.0
 plone.app.drafts = 2.0.0
 plone.app.jquerytools = 1.9.5
-plone.app.mosaic = 3.0.3
-plone.app.standardtiles = 3.1.2
+plone.app.mosaic = 3.1.2
+plone.app.standardtiles = 3.1.1
```

There should be no reason to downgrade plone.app.standardtiles.  I guess this was an accidental change, or it went wrong with some merges.

```suggestion
plone.app.standardtiles = 3.1.2
```